src/components/campaigns/Summary/index.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import moment from 'moment'
+import TimeToGive from './TimeToGive'
+import FundsRaised from './FundsRaised'
+import NumberOfDonations from './NumberOfDonations'
+import Goal from './Goal'
+import GoalProgress from './GoalProgress'
+import totals from '../../../api/totals'
+import campaigns from '../../../api/campaigns'
+
+export default React.createClass({
```

Comment:
Not fussed but we should probably use ES6 class syntax for stateful components -shrug-